src/Tribe/Template.php

```diff
@@ -519,19 +533,32 @@ public function template( $name, $context = array(), $echo = true ) {
 		// Setup the Hook name
 		$hook_name = implode( '/', $namespace );
 
-		// Check if the file exists
-		$file = $this->get_template_file( $name );
+		// Cache file location and existence.
+		if ( ! isset( $file_exists[ $cache_name_key ] ) || ! isset( $files[ $cache_name_key ] ) ) {
+			// Check if the file exists
+			$files[ $cache_name_key ] = $file = $this->get_template_file( $name );
```

Comment:
:no_entry_sign: Assignments must be the first block of code on a line

<sub>hash: 3d5b6c8c2df5e6c4142f790d41d78268</sub>